widgetry/src/widgets/checkbox.rs

```diff
@@ -11,204 +10,194 @@ pub struct Checkbox {
-                .color(RewriteColor::ChangeAll(color.alpha(0.3)))
-                .batch()
-                .centered_vert(),
-            label.batch_text(ctx),
-        ])
-        .to_geom(ctx, None);
-        let (on, _) = Widget::row(vec![
-            GeomBatch::from_svg_contents(include_bytes!("../../icons/checkbox.svg").to_vec())
-                .color(RewriteColor::Change(Color::BLACK, color))
-                .batch()
-                .centered_vert(),
-            label.batch_text(ctx),
-        ])
-        .to_geom(ctx, None);
+        let buttons = ctx.style().btn_plain_light().label_text(label).padding(4.0);
+
+        let false_btn = buttons
+            .clone()
+            .image_path("../widgetry/icons/checkbox_unchecked.svg")
+            .image_color(
+                RewriteColor::Change(Color::BLACK, color.alpha(0.3)),
+                ControlState::Default,
+            );
+
+        let true_btn = buttons
+            .image_path("../widgetry/icons/checkbox_checked.svg")
+            .image_color(
+                RewriteColor::Change(Color::BLACK, color),
+                ControlState::Default,
+            );
 
         Checkbox::new(
             enabled,
-            Btn::custom(
-                off.clone(),
-                off.color(RewriteColor::Change(
-                    Color::WHITE,
-                    ctx.style().hovering_color,
-                )),
-                hitbox.clone(),
-                None,
-            )
-            .build(ctx, label, None),
-            Btn::custom(
-                on.clone(),
-                on.color(RewriteColor::Change(
-                    Color::WHITE,
-                    ctx.style().hovering_color,
-                )),
-                hitbox,
-                None,
-            )
-            .build(ctx, label, None),
+            false_btn.build(ctx, label),
+            true_btn.build(ctx, label),
         )
         .named(label)
     }
 
     // TODO These should actually be radio buttons
-    pub fn toggle<I: Into<String>, MK: Into<Option<MultiKey>>>(
+    pub fn toggle<MK: Into<Option<MultiKey>>>(
         ctx: &EventCtx,
-        label: I,
-        left_label: I,
-        right_label: I,
+        label: &str,
+        left_label: &str,
+        right_label: &str,
         hotkey: MK,
         enabled: bool,
     ) -> Widget {
-        let left_label = left_label.into();
-        let right_label = right_label.into();
-        let hotkey = hotkey.into();
-        let right =
-            GeomBatch::from_svg_contents(include_bytes!("../../icons/toggle_right.svg").to_vec());
-        let left =
-            GeomBatch::from_svg_contents(include_bytes!("../../icons/toggle_left.svg").to_vec());
-        let hitbox = right.get_bounds().get_rectangle();
+        let mut toggle_left_button = ctx
+            .style()
+            .btn_plain_light_icon("../widgetry/icons/toggle_left.svg")
```

Comment:
I had switched to `include_bytes!` recently in order to simplify how other repositories can depend on widgetry. They shouldn't have to keep around a `data/` directory with the right structure. https://github.com/dabreegster/streetcad is a concrete test case of this. Could we go back to `include_bytes!` or similar for all of the widgetry SVGs? Also fine to do that in a later PR all at once, since there are a few here.

Reply:
Ah, that makes total sense. Would you mind if I addressed this in a follow-up PR? I have some backlogged commits, and I'm hoping to avoid some de-conflicting.

Reply:
Yup, no rush!